### ISSUE TITLE
list loadbalancer_certificate_arn 

### ISSUE THREAD

**Issue by mrl045** [cloudposse/terraform-aws-elastic-beanstalk-environment#47]
Hi Guys,

Thank you for the great work you made available. Currently I am bumped to this particularly use case:
Using Multi-Docker environments which are reachable through an Application Loadbalancer with HostHeaders enabled. Some of the applications are reachable through other HostHeaders: e.g. app.domain1.com and web.domain2.com. 

Terraform recently presented a fix for the AWS provider which allows usage of multiple ACM certificates for an Application Loadbalancer (https://github.com/terraform-aws-modules/terraform-aws-alb/issues/26)
<img width="709" alt="capture" src="https://user-images.githubusercontent.com/19322254/45368030-e3918000-b5e2-11e8-90da-536ecadc1dd3.PNG">
. 

TL:DR is it possible to change loadbalancer_certificate_arn to type of List. 

`loadbalancer_certificate_arn              = ["${module.acm-application.acm_certificate_arn}"]`

**Comment by osterman** [comment]
@mrl045 sounds good to me. Would you be able to submit a PR?

**Comment by osterman** [comment]
Btw, join our slack community if you haven't already: https://slack.cloudposse.com

**Comment by aknysh** [comment]
@mrl045 thanks for opening the issue?
will you be able to test and submit a PR?
if not for any reason, we'll probably take a look in a few days

**Comment by SubrahmanyaSarma** [comment]
Hi All,
I am also facing the same issue. have you found the solution for this scenario.  Thanks.
